Issue #407: Register Endpoint as Windows AV (#451)
elastic/security-docs: docs/getting-started/install-endpoint.asciidoc

[role="screenshot"]
image::images/install-endpoint/malware-protection.png[]

## Register as antivirus (Windows Only)

> If you download the Elastic Agent on Windows 10 or above, you can Elastic Security as your antivirus software. To enable Elastic Security as antivirus: 

Reviewer: ```suggestion
If you download the Elastic Agent on Windows 10 or above, you can configure Elastic Security as your antivirus software by doing the following:
```

Reviewer: Consider changing `download` -> `downloaded` because they likely will have done so by the time they get to this step.

1. Go to **Fleet** > **Agents**. A list of hosts with the Elastic Agent appears. 
2. Select your desired host from the list.

> 3. From the host's page, find your named **Endpoint** integration and select it. 

Reviewer 2: This section makes it sound like the register as AV option will only apply to a selected host.  When you turn the option on, any Windows 10 Host with that particular Agent Policy will receive the "Register as AV" option.

I would drive the user towards choosing a Policy and informing then that any host using that Agent Policy would receive the Register as AV option

Author: Kevin: The intention will be to move these steps into an overall Policy overview once this merged in. I think that's help set the context better.

4. On the Integration Edit page, look for the **Protections** section and find **Type: Register as antivirus**. Toggle this option to enable. 
+
[role="screenshot"]
image::images/register-as-antivirus.png[]

> 5. Click **Save Integration**. Confirm that you would like to update your Elastic Agent. Once updated, Windows defender will be disabled on your local host in favor of Elastic Security.

Reviewer: ```suggestion
5. Click **Save Integration**. Confirm that you would like to update your Elastic Agent. Once updated, Windows Defender will be disabled on your local host in favor of Elastic Security.
```
